### PR TITLE
Improve English translations

### PR DESCRIPTION
**Pull request by liutikas**

### CODE REVIEW

File: airrohr-firmware/intl_en.h [hunk 1]
```diff
@@ -48,15 +48,15 @@ const char INTL_LCD1602_3F[] PROGMEM = "LCD 1602 (I2C: 0x3F)";
 const char INTL_LCD2004_27[] PROGMEM = "LCD 2004 (I2C: 0x27)";
 const char INTL_DISPLAY_WIFI_INFO[] PROGMEM = "Display Wifi info";
 const char INTL_DISPLAY_DEVICE_INFO[] PROGMEM = "Display device info";
-const char INTL_DEBUG_LEVEL[] PROGMEM = "Debug&nbsp;Level";
-const char INTL_MEASUREMENT_INTERVAL[] PROGMEM = "Measuring interval";
+const char INTL_DEBUG_LEVEL[] PROGMEM = "Debug level";
```
Comment (dirkmueller) [inline airrohr-firmware/intl_en.h]: why did you remove the nonbreaking space here?

Comment (liutikas) [reply airrohr-firmware/intl_en.h]: Some of the other translations do not have it. Almost every other string does not contain it. From local testing it renders the same way without it.

Comment (liutikas) [reply airrohr-firmware/intl_en.h]: Would you like me to undo this part of the change?

Comment (dirkmueller) [reply airrohr-firmware/intl_en.h]: I looked a bit at this. the nbsp isn't strictly needed as of today because there is INTL_DURATION_ROUTER_MODE in the same table column with nbsp, and the translation of that one is almost certainly going to be the longest. 

Maybe you could fix up the 5 other cases that do not use &nbsp to make it consistent (most of them do use it).

Comment (dirkmueller) [reply airrohr-firmware/intl_en.h]: I fixed this now in #594. thanks for the contribution!
